src/Bicep.Cli.IntegrationTests/BuildCommandTests.cs

```diff
@@ -320,13 +316,9 @@ public async Task Build_WithInvalidBicepConfig_ShouldProduceOutputFile()
 
             var (output, error, result) = await Bicep("build", inputFile);
 
-            result.Should().Be(0);
+            result.Should().Be(1);
             output.Should().BeEmpty();
-            error.Should().BeEmpty();
-
-            var expectedOutputFile = Path.Combine(testOutputPath, "main.json");
-
-            File.Exists(expectedOutputFile).Should().BeTrue();
+            error.Should().StartWith($"Could not load the bicep configuration file \"{configurationPath}\". Expected depth to be zero at the end of the JSON payload.");
```

Comment:
I know this is just coming from the library, but "Expected depth to be zero at the end of the JSON payload." feels more confusing to the end user than just saying something generic like "Failed to read the file contents as valid JSON"

Reply:
The error message is now formatted as shown below (similar to Marcin's example, but more verbose):
```
Failed to parse the contents of the Bicep configuration file "{configPath}" as valid JSON: "{message}".
```